backend/templates/battling/settled_battles.html

```diff
@@ -0,0 +1,37 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Poke Battle</title>
+  <style>
+    body {
+      background: #EFEFBB;
+      background: linear-gradient(to right, rgb(197, 230, 236), rgb(239, 187, 230));
+      color: white;
+      margin: 0;
+      font-family: rubik;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+    <h1>Settled Battles</h1>
+    {% if object_list.all %}
+    {% for instance in object_list %}
```

Comment:
Let's keep the same pattern of indentation on the templates

Reply:
I try, but when I do indentation, vs code itself changes it however it wants to :(

Reply:
The ideal is to disable whatever it is updating the templates